skglm/estimators.py

```diff
@@ -959,19 +959,28 @@ class SparseLogisticRegression(LinearClassifierMixin, SparseCoefMixin, BaseEstim
 
     The optimization objective for sparse Logistic regression is:
 
-    .. math:: 1 / n_"samples" sum_(i=1)^(n_"samples") log(1 + exp(-y_i x_i^T w))
-        + alpha ||w||_1
+    .. math:: 
+        \frac{1}{n_{\text{samples}}} \sum_{i=1}^{n_{\text{samples}}}
+        \log\left(1 + \exp(-y_i x_i^T w)\right)
+        + \alpha \cdot \left( \text{l1_ratio} \cdot \|w\|_1 +
```

Comment:
go without the cdots for lighter formulas; also the squared L2 term should be divided by 2 (check the Enet docstring)